src/components/Select/AutosuggestMulti/SuggestionTag/__tests__/SuggestionTag.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { SuggestionTag } from '../SuggestionTag';
+
+describe('SuggestionTag', () => {
+    it('should render correctly', () => {
+        const wrapper = mount(<SuggestionTag>tag</SuggestionTag>);
+
+        expect(toJson(wrapper)).toMatchSnapshot();
+        expect(wrapper.find('button')).toHaveLength(0);
+    });
+    it('should render correctly with styles modifier', () => {
+        const wrapper = mount(
+            <SuggestionTag isBounded isStretched>
```

Comment:
does it actually make sense to have both this props? I feel this makes the API confusing and you can easily misuse it, no?

Reply:
already discussed in Slack